### PR TITLE
Added info how to update existing environment to README. First try to…

### PR DESCRIPTION
**Pull request by nicohoermann**
…wards fixing the linking issue #4. Seems to work locally.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,14 @@ conda config --set channel_priority strict
...
 ```
 
+Alternatively, if you want to update an existing environment use:
+
+```
+conda env update --name <your_env_name> --file environment.yml
+```
+
+
+
```
Comment (saraedum) [inline README.md:22]:
```suggestion
Alternatively, if you want to install the required packages into an existing, environment use:

```
conda env update --name <your_env_name> --file environment.yml
```

```

File: echemdb/make_pages.py [hunk 2]
```diff
@@ -78,7 +78,7 @@ def get_page_links( property_propertyvals_dict ):
             filename = get_echemdb_id_file(echemdb_id)
             ln = f'{echemdb_id}'
 
-    return f'[{ln}](../../../{filename})'
+    return f'[{ln}](../../{filename}/)'
```
Comment (saraedum) [inline echemdb/make_pages.py:81]: I would not be surprised if this breaks some other links. But anyway, let's try :)

**Review by saraedum — APPROVED** [review]

**Comment by saraedum** [comment]
(@nicohoermann you did not set this to "ready for review". I hope you don't mind that I had a look anyway.)

File: echemdb/make_pages.py (hunk 2 above)
Comment (nicohoermann) [reply echemdb/make_pages.py:81]: Yeah, could be just wanted to see if it helps something. At least I got back a bit into the code...

**Pull request merged**